local-volume/README.md

```diff
@@ -37,11 +37,19 @@ What doesn't work and workarounds:
         * Give your pods high priority.
         * Run a workaround controller that unbinds PVCs for pods that are
           stuck pending. TODO: add link
-* External provisioner cannot correctly detect capacity of mounts added after it
-  has been started.
-    * This requires mount propagation to work, which is targeted for 1.9.
-    * Workaround: Before adding any new mount points, stop the daemonset, add
-      the new mount points, start the daemonset.
+* The provisioner does not correctly detect mounts added after it
+  has been started, and may fail or incorrectly creates local PVs with
+  zero capacity. This situation manifests as a race condition between discovery
```

Comment:
zero capacity -> the root filesystem capacity

Reply:
It's actually not a race condition right now because there is no mount propagation enabled.  It always happens.

Reply:
Ok. Updated.